sdks/python/apache_beam/runners/common.py

```diff
@@ -188,3 +188,35 @@ def counter_for(self, aggregator):
     """Looks up the counter for this aggregator, creating one if necessary."""
     return self._counter_factory.get_aggregator_counter(
         self.step_name, aggregator)
+
+
+class DoFnContext(object):
```

Comment:
Should `core.DoFnContext` be removed? If we are going to have the two for a while could you add a comment to the one that is not used. (https://github.com/apache/incubator-beam/blob/python-sdk/sdks/python/apache_beam/transforms/core.py#L54)